.github/pull_request_template.md

```diff
@@ -0,0 +1,38 @@
+## Changes
+
+_Explanation of changes goes here_
+
+## Notes
+
+_Any other notes go here_
+
+## To Do
```

Comment:
Can you elaborate on why a to-do section is necessary for a pull request? From my understanding, people will make pull requests only when their code is ready to be reviewed, not when there are still things to do for the issue.

Reply:
I think that we should let people open PRs before they're done so that reviews can be done earlier and it's easier to share/check in about it. The todo section would let them have a space to keep track of what they need to do before merging or reviewing, and would also let reviewers know at a glance what hasn't been done yet so they know it isn't getting forgotten.

Reply:
Ah okay, that makes sense.